### PR TITLE
Some adjustements for the first 100 questions

### PR DESCRIPTION
**Pull request by PrzemyslawSwiderski**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -797,11 +797,11 @@ We are so thankful for every contribution, which makes sure we can deliver top-n
 
 **[⬆ Back to Top](#table-of-contents)**
 
-### A developer is creating a serverless web application and maintains different branches of code. The developer wants to avoid updating the Amazon API Gateway target endpoint each time a new code push is performed. What solution would allow me developer toPerform a code push efficiently, without the need to update the API Gateway?
+### cation and maintains different branches of code. The developer wants to avoid updating the Amazon API Gateway target endpoint each time a new code push is performed. What solution would allow the developer to perform a code push efficiently, without the need to update the API Gateway?
```
[the remark this answers is not in the document]
Comment (danieldanielecki) [reply README.md:800]: Fixed in https://github.com/Ditectrev/Amazon-Web-Services-AWS-Developer-Associate-DVA-C02-Practice-Tests-Exams-Questions-Answers/commit/a7f4b430c261bf62f1772c224eab319e87bdf601

File: README.md [hunk 2]
```diff
@@ -1097,36 +1096,26 @@ We are so thankful for every contribution, which makes sure we can deliver top-n
...
-- [x] Use the AWS CLI to get the metrics.
+- [ ] Use the AWS CLI to get the metrics.
 - [ ] Analyze the applications and remove the API call.
-- [ ] Retry the call with exponential backoff.
-
-**[⬆ Back to Top](#table-of-contents)**
-
-### A Developer is receiving HTTP 400: ThrottlingException errors intermittently when calling the Amazon CloudWatch API. When a call fails, no data is retrieved. Which techniques will work? (Choose TWO)
-
-- [ ] Define a Swagger file. Use AWS Elastic Beanstalk to deploy the Swagger file.
```
[the remark this answers is not in the document]
Comment (danieldanielecki) [reply README.md:1125]: There's already such a question - number 380 ;-)

**Review by danieldanielecki — COMMENTED** [review]
Fantastic contribution, much, much appreciated!

File: README.md (hunk 2 above)
[the remark this answers is not in the document]
Comment (PrzemyslawSwiderski) [reply README.md:1125]: right :)

**Pull request merged**